From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Budris <budris@amazon.com>
Date: Thu, 19 Jan 2023 15:33:41 -0800
Subject: [PATCH] remove hardcoded requierment on flannel plugin

In CNI v1.0.0 the flannel plugin was removed; so, when building EKS Distro 1.23
or greater the plugin is not present and cannot be loaded.
However, EKS Distro does not use the flannel plugin in any cases
and the requirement hard-coded in kOps can be safely removed.
---
 nodeup/pkg/model/networking/common.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/nodeup/pkg/model/networking/common.go b/nodeup/pkg/model/networking/common.go
index 0f3b170fc5..e514689151 100644
--- a/nodeup/pkg/model/networking/common.go
+++ b/nodeup/pkg/model/networking/common.go
@@ -34,7 +34,6 @@ func (b *CommonBuilder) Build(c *fi.NodeupModelBuilderContext) error {
 	assets := []string{
 		"bridge",
 		"dhcp",
-		"flannel",
 		"host-device",
 		"host-local",
 		"ipvlan",
